bitnami/ghost/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: ghost
-version: 10.1.20
+version: 10.1.21
```

```suggestion
version: 10.2.0
```

As we are adding a new parameter, we should bump the minor version instead.